emcc.py

```diff
@@ -2092,7 +2092,7 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
         # BINARYEN_METHOD with something that doesn't use the polyfill, then we don't need it.
         if not shared.Settings.BINARYEN_METHOD or 'interpret' in shared.Settings.BINARYEN_METHOD:
           logging.debug('integrating wasm.js polyfill interpreter')
-          wasm_js = open(os.path.join(binaryen_bin, 'wasm.js')).read()
+          wasm_js = open(os.path.join(shared.Settings.BINARYEN_ROOT, 'share', 'binaryen', 'wasm.js')).read()
```

Comment:
a recent binaryen change altered where we install files. maybe that's the difference here. i think it used /usr/share. but here we want the source dir i think, not the install dir